openmdao/core/vec_wrapper_complex_step.py

```diff
@@ -0,0 +1,298 @@
+""" VecWrapper 'wrapper' that is used for component-wise complex step."""
+
```

Comment:
We could consider it, but it seems like we are trying to streamline our vectors right now, so i didn't want to add any very situational stuff to it.